app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -846,6 +847,16 @@ private void enableActionMode(int inputPosition) {
         toggleSelection(inputPosition);
     }
 
+    private void scaleScreen() {
+        DisplayMetrics displayMetrics = new DisplayMetrics();
+        getWindowManager().getDefaultDisplay().getMetrics(displayMetrics);
+        int screenHeight = displayMetrics.heightPixels;
+        float mediumSizePx = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP,MEDIUM_SCALE_FACTOR_DIP,getResources().getDisplayMetrics());
+        boolean shouldScaleSmaller = screenHeight < mediumSizePx;
+
+        binding.include.welcomeIcon.setVisibility(shouldScaleSmaller ? View.GONE : View.VISIBLE);
```

Comment:
Didn't notice this before but it seems this crashes the archive option (you can confirm this by putting 1 card in archive and then clicking the menu in the top right and choosing "Archive"):

```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: me.hackerchick.catima.debug, PID: 19126
    java.lang.RuntimeException: Unable to resume activity {me.hackerchick.catima.debug/protect.card_locker.MainActivity}: java.lang.NullPointerException: Attempt to read from field 'protect.card_locker.databinding.ContentMainBinding protect.card_locker.databinding.MainActivityBinding.include' on a null object reference
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4824)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4857)
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54)
        at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2253)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7870)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
     Caused by: java.lang.NullPointerException: Attempt to read from field 'protect.card_locker.databinding.ContentMainBinding protect.card_locker.databinding.MainActivityBinding.include' on a null object reference
        at protect.card_locker.MainActivity.scaleScreen(MainActivity.java:857)
        at protect.card_locker.MainActivity.onResume(MainActivity.java:407)
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1488)
        at android.app.Activity.performResume(Activity.java:8197)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4814)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4857) 
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54) 
        at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2253) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loopOnce(Looper.java:201) 
        at android.os.Looper.loop(Looper.java:288) 
        at android.app.ActivityThread.main(ActivityThread.java:7870) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003) 
```

Reply:
Ooops.. I didn't think about this scenario. It's now fixed in commit [b0008cd](https://github.com/CatimaLoyalty/Android/pull/1095/commits/b0008cd639687a21bece7ea4ec5d2beb31ba570b)